adapters/handlers/grpc/v1/parse_search_request.go

```diff
@@ -804,6 +803,12 @@ func extractAdditionalPropsFromMetadata(class *models.Class, prop *pb.MetadataRe
 		Vectors:            prop.Vectors,
 	}
 
+	if vectorSearch && configvalidation.CheckCertaintyCompatibility(class, targetVectors) != nil {
+		props.Certainty = false
+	} else {
+		props.Certainty = prop.Certainty
+	}
```

Comment:
`prop.Certainty` will always be false here, right? Since requests can only set `Certainty` within vector searches so the default will pass through otherwise?

Reply:
hmm no? There is a test for that in `adapters/handlers/grpc/v1/parse_search_request_test.go::1673` :)

We will disable it for vector search AND incompatible class settings and otherwise use the userinput